protocol/x/clob/e2e/withdrawal_gating_test.go

```diff
@@ -252,7 +252,10 @@ func TestWithdrawalGating_NegativeTncSubaccount_BlocksThenUnblocks(t *testing.T)
 					tApp.App.SubaccountsKeeper.GetSubaccount(ctx, *expectedSubaccount.Id),
 				)
 			}
-			negativeTncSubaccountSeenAtBlock, exists := tApp.App.SubaccountsKeeper.GetNegativeTncSubaccountSeenAtBlock(ctx)
+			negativeTncSubaccountSeenAtBlock, exists := tApp.App.SubaccountsKeeper.GetNegativeTncSubaccountSeenAtBlock(
+				ctx,
+				satypes.ModuleAddress,
```

Comment:
nit: can we use a hardcoded constant `DEFAULT_CROSS_MARGIN_KEY` instead of the module address? i don't think we would change the subaccount module name but it could in the future

Reply:
just in case the subaccount module name changes, no need for a state migration. up to you

Reply:
Changed to a set of suffixes including a hard-coded constant for cross-perpetuals.